fmn/consumer/backend.py

```diff
@@ -206,11 +206,11 @@ def read(queue_object):
     for recipient in recipients:
         user = recipient['user']
         t = time.time()
-        pref = PREFS.get('%s__%s' % (user, context))
+        pref = PREFS.get('%s_%s' % (user, context))
```

Comment:
This was a silly thing I managed to somehow typo (obviously there's no test coverage) when converting stuff. The dictionary uses the key format fixed here.